core/src/components/datetime/test/basic/index.html

```diff
@@ -394,7 +394,7 @@ <h2>Modal - Custom</h2>
           // create component to open
           const element = document.createElement('div');
           element.innerHTML = `
-            <ion-datetime show-default-buttons="true" color="danger">
+            <ion-datetime id="datetime-modal" show-default-buttons="true" color="danger">
```

Comment:
Add `value="2023-01-01"` to the `ion-datetime` and that should help create stable screenshots for your test (you will need to re-generate the reference screenshots with the Github Action after committing the change).

Currently it is always loading the "latest date/time" when the modal is presented, which will produce a failed test for any time in the future.

Reply:
Thank you!!! Once again, screenshots making it difficult.